docs/specifications/tests/Zone-TP/zone09.md

```diff
@@ -37,37 +37,37 @@ that the well known mailbox is not expected for reverse zones
 (in-addr.arpa, ip6.arpa). Those are excluded from the requirement in
 this test case.
 
-The root zone cannot be a mail domain since the mail domain is
+The root zone cannot be an email domain since the email domain is
 the part to the left of the trailing dot, e.g. "example.com" is the
-the maildomain for tha apex of "example.com.". The root zone is
+email domain for the apex of "example.com.". The root zone is
 excluded from the requirement.
 
-Top-level domains (TLDs) can technically function as mail domains
-([RCF 5321][RFC 5321#section-2.3.5], section 2.3.5) but they rarely
-have that function and are probably not meant to be included in the
-specification in [RFC 2142]. On the contrary,
-[Internet Architecture Board] concludes in a report
-"[Dotless Domains Considered Harmful][IAB Statement]" that TLD names
-should not be mail domains. In this test case TLDs are not only
-excluded from the requirement of being an mail domain, if found to
-be, a warning will be generated that points that out.
-
-It is therefore expected for every domain (zone), excluding those
-excepted above, to publish an MX record in apex of the zone (i.e.
-in the same position as the SOA record). If MX is not present, SMTP
-*can* deliver email using an address record (A or AAAA) as specified
-in [RFC 5321][RFC 5321#section-5.1], section 5.1, but that possibility
-is not in common use. This test case only checks for MX record since
-that is the absolutely dominating way to configure support for email.
-If an address record is found in apex it is more often used for web
-service rather than mail service.
+Top-level domains (TLDs) can technically function as email domains
+([RCF 5321][RFC 5321#section-2.3.5], section 2.3.5) but they rarely have that
+function and are probably not meant to be included in the specification in
+[RFC 2142]. On the contrary, [Internet Architecture Board] concludes in a report
+"[Dotless Domains Considered Harmful][IAB Statement]" that domain names that only
+consists of one label, e.g. "se", "fr" or "com", should not be used for various
+Internet services. This means TLD names should not be used as email domains.
+In this test case TLDs are not only excluded from the requirement of being an
+email domain, if found to be, a warning will be generated that points that out.
+
+It is therefore expected for every domain (zone), excluding those above, to
+publish an MX record in apex of the zone (i.e. in the same position as the SOA
+record). If MX is not present, SMTP *can* deliver email using an address record
+(A or AAAA) as specified in [RFC 5321][RFC 5321#section-5.1], section 5.1, but
+that possibility is not in common use. This test case only checks for MX record
+and ignores the possibility to use address records for email since that is the
+absolutely dominating way to configure support for email is to use MX. If an
```

Comment:
too many words I guess: "since that is the absolutely ... is to use MX" -> "since the absolutely ... is to use MX"

Reply:
Yes. Fixed.